doc/source/toolchain.rst

```diff
@@ -0,0 +1,226 @@
+Toolchain Roadmap
+=================
+
+The use of the SciPy library requires (or optionally depends upon) several
+other libraries in order to operate, the main dependency being Python and NumPy.
+It requires a larger collection of libraries and tools in order to build the library,
+or to build the documentation.
+
+Of course, the tooling and libraries are themselves not static.  This document aims to provide a
+guide as to how SciPy's use of these dynamic dependencies will proceed over time.
+
+SciPy aims to be compatible with a number of releases of its dependent libraries and tools.
+Forcing the user base to other components for upgrade for every release would greatly
+diminish the value of SciPy.  However, maintaining backwards compatibility
+with very old tooling/libraries imposes limitations on which newer functionalities and capabilities
+can be incorporated.  SciPy takes a somewhat conservative approach, maintaining compatibility with
+several major releases of Python and NumPy on the major platforms. (That may in of itself impose
+further restrictions.  See the C Compilers section for an example.)
+
+
+- First and foremost, SciPy is a Python project hence it requires a Python environment.
+- The LAPACK and OpenBLAS numerical libraries need to be installed.
```

Comment:
"The LAPACK and OpenBLAS" --> "BLAS and LAPACK" ... any library conforming to the BLAS/LAPACK interface may be used; OpenBLAS, ATLAS, MKL, BLIS and reference Netlib libraries are known to work.

Reply:
Adde the longer text to the "Other Libraries" section.